modules/lib/core/index.d.ts

```diff
@@ -19,6 +21,132 @@ declare global {
     }
 }
 
+//──────────────────────────────────────────────────────────────────────────────
+// Types that are imported by @enonic-types/global
+//──────────────────────────────────────────────────────────────────────────────
+export interface App {
+    /**
+     * The name of the application.
+     *
+     * @type string
+     */
+    name: string;
```

Comment:
I think we should define all these fields (in global variables) as `readonly`. Do you have any objections? Did I miss something, and did we have a reason not to do so?

Reply:
Dunno, seems outside what I'm trying to do in this branch anyway.